app/helpers/users_helper.rb

```diff
@@ -62,20 +48,87 @@ def redirect_back_or(default)
   def store_location
     session[:forwarding_url] = request.original_url if request.get?
   end
+  def is_admin?
+    if !current_user.nil? &&current_user.admin
+      return true
+    end
+    return false
 
+  end
 
-  # Returns true if the user is logged in, false otherwise.
   def is_access?
```

Comment:
@chandu1307  Method name should always be more readable. 
You can pass review as parameters and then check if that review belongs to currenct_user.

def can_user_access_review? review
     if currenct_user.try(:id) == review.user_id
           return true
      else
          return false
      end
end